proto/backend.proto

```diff
@@ -155,6 +155,9 @@ message DataResponse {
   // The status codes should match values from standard HTTP status codes
   // If not set explicitly, it will be marshaled to 200 if no error exists, or 500 if one does
   int32 status = 4;
+
+  // Error source
+  string source = 5;
```

Comment:
Here the name is `source` and elsewhere `ErrorSource`. Suggest to rename it to `errorSource` here for consistency/clarity. 

```suggestion
  // Error source identifies from where an error originates. e.g. plugin or downstream.
  string errorSource = 5;
```

Reply:
updated